rebar.lock

```diff
@@ -0,0 +1 @@
+[].
```

Comment:
doesn't rebars add plugins version into the lock file?

Reply:
noppp.